test/python/WMCore_t/Misc_t/Runtime_t.py

```diff
@@ -155,7 +155,7 @@ def createTestWorkload(self, workloadName='Test', emulator=True):
             os.makedirs(siteConfigPath)
         shutil.copy('site-local-config.xml', siteConfigPath)
         environment = rereco.data.section_('environment')
-        environment.CMS_PATH = workloadDir
+        environment.SITECONFIG_PATH = workloadDir
```

Comment:
Shouldn't `SITECONFIG_PATH` be:

```
environment.SITECONFIG_PATH = "%s/SITECONF/local" % workloadDir
```
To make it consistent with the default definition here:
https://github.com/cms-sw/cms-common/blob/9c8203bf637b153697b855dfa522d6741379bcc9/cmsset_default.sh#L40-L44

Reply:
I am not sure from the L153 above. Maybe it even does not really matter, but let me make this change.